test/TestProjects/MgmtScopeResource/Generated/Extensions/MgmtScopeResourceExtensions.cs

```diff
@@ -23,31 +23,43 @@ public static partial class MgmtScopeResourceExtensions
     {
         private static MockableMgmtScopeResourceArmClient GetMockableMgmtScopeResourceArmClient(ArmClient client)
         {
+            Argument.AssertNotNull(client, nameof(client));
```

Comment:
these validations are added.
The invocation will fail anywhere if the client is null - it will throw `NullReferenceException` which might confuses our customers.